### PR TITLE
Set cargo description and license

### PR DESCRIPTION
**Pull request by jakelee8**

### CODE REVIEW

File: bitski-local/Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 name = "bitski-local"
 version = "0.1.0"
 edition = "2021"
+publish = false
```
Comment (andschneider) [inline bitski-local/Cargo.toml:5]: why isn't this published?

Comment (jakelee8) [reply bitski-local/Cargo.toml:5]: Cargo won't publish it because it has a git dependency, so this flag is just a sanity check

Comment (jakelee8) [reply bitski-local/Cargo.toml]: Removed b/c it's redundant

Comment (andschneider) [reply bitski-local/Cargo.toml]: ah yeah I forgot about the git dependency

**Review by andschneider — APPROVED** [review]

**Pull request merged**